app/javascript/dashboard/components-next/message/Message.vue

```diff
@@ -182,17 +184,19 @@ const isMyMessage = computed(() => {
   ) {
     return true;
   }
+
+  if (props.messageType === MESSAGE_TYPES.ACTIVITY) {
+    return false;
+  }
+
   const senderId = props.senderId ?? props.sender?.id;
   const senderType = props.senderType ?? props.sender?.type;
 
   if (!senderType || !senderId) {
-    return false;
+    return true;
   }
 
-  return (
-    senderType.toLowerCase() === SENDER_TYPES.USER.toLowerCase() &&
-    props.currentUserId === senderId
-  );
+  return senderType.toLowerCase() === SENDER_TYPES.USER.toLowerCase();
```

Comment:
Any reason to remove the `userId` check?

Reply:
@scmmishra Removed the `userId` check because it was displaying other agents messages on the left side of the chat. Since we are moving all messages to the right side except those from the contact.